src/Uno.Toolkit.UI/Controls/Chips/ChipGroup.cs

```diff
@@ -271,12 +270,18 @@ private void SynchronizeInitialSelection()
 
 			if (SelectedItem != null)
 			{
-				OnSelectedItemChanged(null);
+				// TODO: Something looks wrong here.
+				// OnSelectedItemChanged doesn't expect null.
+				// It either won't do something useful, or will crash with NullReferenceException.
+				OnSelectedItemChanged(null!);
```

Comment:
This should be fixed properly. @Xiaoy312 @kazo0

Reply:
@jeromelaban The existing behavior is either a crash, or the call doing nothing. I could remove the call entirely and nothing will regress. But I preferred to suppress and let someone else take a look in case there is a real issue here.